NGCHM/WebContent/javascript/Linkout.js

```diff
@@ -645,17 +645,21 @@ var linkoutsVersion = 'undefined';
 				LNK.addMenuItemToTable(axis, table, grpLinkouts[l].linkout, true);
 			}
 		} else {
-			//Always add clipboard link at top of list
-			LNK.addMenuItemToTable(axis, table, grpLinkouts[0].linkout, true);
+			const covar = axis.indexOf('Covar') != -1;
```

Comment:
i.e. axis is RowCovar or ColumnCovar.  A future rewrite of Linkout.js should eliminate magic name strings.